kinesumer.go

```diff
@@ -222,7 +222,7 @@ func (kin *Kinesumer) Begin() ([]*ShardWorker, error) {
 
 	kin.Options.ErrHandler(NewError(EInfo, fmt.Sprintf("%v/%v workers started", kin.nRunning, n), nil))
 
-	return workers, nil
```

Comment:
There's pretty much nothing useful you can do with the workers, and this sort of simplifies a dependency cycle without introducing an interface for the worker objects. (I want /kinesumer/mocks to not depend on /kinesumer, because tests in /kinesumer use the mocks.)